App/Containers/ThreadsList.js

```diff
@@ -119,7 +120,8 @@ class ThreadsList extends React.PureComponent {
 
   _onPressItem = (photo) => {
     const { id, name } = photo
-    this.props.viewThread(id, name)
+    this.props.viewThread(id)
+    this.props.navigation.navigate('ViewThread')
```

Comment:
And this just simply navigates to the thread details screen. That screen gets it's data from redux only.